# Persian translation for glib-networking.
# Copyright (C) 2011 glib-networking's COPYRIGHT HOLDER
# This file is distributed under the same license as the glib-networking package.
# Arash Mousavi <mousavi.arash@gmail.com>, 2011, 2013.
# Danial Behzadi <dani.behzi@ubuntu.com>, 2020.
#
msgid ""
msgstr ""
"Project-Id-Version: glib-networking master\n"
"Report-Msgid-Bugs-To: https://gitlab.gnome.org/GNOME/glib-networking/issues\n"
"POT-Creation-Date: 2021-06-25 16:32+0000\n"
"PO-Revision-Date: 2021-09-01 16:50+0430\n"
"Last-Translator: Danial Behzadi <dani.behzi@ubuntu.com>\n"
"Language-Team: Persian\n"
"Language: fa\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Poedit-SourceCharset: utf-8\n"
"X-Generator: Poedit 2.4.2\n"
"Plural-Forms: nplurals=2; plural=(n==0 || n==1);\n"

#: proxy/libproxy/glibproxyresolver.c:164
msgid "Proxy resolver internal error."
msgstr "خطای داخلی تحلیل‌گر پیشکار."

#: tls/base/gtlsconnection-base.c:562 tls/base/gtlsinputstream.c:78
#: tls/base/gtlsinputstream.c:141 tls/base/gtlsoutputstream.c:78
#: tls/base/gtlsoutputstream.c:143
msgid "Connection is closed"
msgstr "اتصال بسته شده است"

#. Cannot perform a blocking operation during a handshake on the
#. * same thread that triggered the handshake. The only way this can
#. * occur is if the application is doing something weird in its
#. * accept-certificate callback. Allowing a blocking op would stall
#. * the handshake (forever, if there's no timeout). Even a close
#. * op would deadlock here.
#.
#: tls/base/gtlsconnection-base.c:636
msgid "Cannot perform blocking operation during TLS handshake"
msgstr "نمی‌توان در طول دست‌دادن TLS عملیات انسداد انجام داد"

#: tls/base/gtlsconnection-base.c:701 tls/base/gtlsconnection-base.c:1249
#: tls/openssl/gtlsconnection-openssl.c:358
msgid "Socket I/O timed out"
msgstr "مهلت سوکت و/خ به سر رسید"

#: tls/base/gtlsconnection-base.c:875
msgid "Server required TLS certificate"
msgstr "کارگزار به گواهینامه TLS احتیاج دارد"

#: tls/base/gtlsconnection-base.c:1451
#, c-format
msgid "Handshake is not finished, no channel binding information yet"
msgstr ""

#: tls/base/gtlsconnection-base.c:1530
msgid "Peer does not support safe renegotiation"
msgstr "جفت از بازمذاکرهٔ امن پشتیبانی نمی‌کند"

#: tls/base/gtlsconnection-base.c:1678 tls/gnutls/gtlsconnection-gnutls.c:482
#: tls/openssl/gtlsconnection-openssl.c:209
#: tls/openssl/gtlsconnection-openssl.c:834
#, c-format
msgid "Unacceptable TLS certificate"
msgstr "گواهینامه TLS غیر قابل پذیرش"

#: tls/base/gtlsconnection-base.c:2152
#, c-format
msgid "Receive flags are not supported"
msgstr "پرچم‌های دریافت پشتیبانی نمی‌شوند"

#: tls/base/gtlsconnection-base.c:2299
#, c-format
msgid "Send flags are not supported"
msgstr "پرچم‌های ارسال پشتیبانی نمی‌شوند"

#: tls/gnutls/gtlscertificate-gnutls.c:361
#: tls/openssl/gtlscertificate-openssl.c:362
#, c-format
msgid "Could not parse DER certificate: %s"
msgstr "تجزیه گواهینامه DER امکان‌پذیر نبود: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:382
#: tls/openssl/gtlscertificate-openssl.c:382
#, c-format
msgid "Could not parse PEM certificate: %s"
msgstr "تجزیه گواهینامه PEM امکان‌پذیر نبود: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:405
#: tls/openssl/gtlscertificate-openssl.c:401
#, c-format
msgid "Could not parse DER private key: %s"
msgstr "تجزیه کلید خصوصی DER امکان‌پذیر نبود: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:428
#: tls/openssl/gtlscertificate-openssl.c:420
#, c-format
msgid "Could not parse PEM private key: %s"
msgstr "تجزیه کلید خصوصی PEM امکان‌پذیر نبود: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:455
#, c-format
msgid "Could not import PKCS #11 certificate URI: %s"
msgstr "نتوانست نشانی گواهی‌نامهٔ ‪PKCS #11‬ را درون‌ریز کند: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:497
#: tls/openssl/gtlscertificate-openssl.c:455
msgid "No certificate data provided"
msgstr "هیچ اطلاعات گواهینامه‌ای ارائه نشده"

#: tls/gnutls/gtlscertificate-gnutls.c:806
#: tls/gnutls/gtlsconnection-gnutls.c:1033
#, c-format
msgid "Cannot verify peer identity of unexpected type %s"
msgstr ""

#: tls/gnutls/gtlsconnection-gnutls.c:198
msgid "Could not create TLS connection:"
msgstr "نتوانست اتّصال TLS ایجاد کند:"

#: tls/gnutls/gtlsconnection-gnutls.c:208 tls/gnutls/gtlsconnection-gnutls.c:228
#: tls/openssl/gtlsclientconnection-openssl.c:458
#: tls/openssl/gtlsserverconnection-openssl.c:503
#, c-format
msgid "Could not create TLS connection: %s"
msgstr "نمی‌توان اتصال TLS ایجاد کرد: %s"

#: tls/gnutls/gtlsconnection-gnutls.c:426 tls/gnutls/gtlsconnection-gnutls.c:437
#: tls/gnutls/gtlsconnection-gnutls.c:451
#: tls/openssl/gtlsconnection-openssl.c:176
#, c-format
msgid "Peer failed to perform TLS handshake: %s"
msgstr "جفت در انجام دست دادن TLS شکست خورد: %s"

#: tls/gnutls/gtlsconnection-gnutls.c:459
msgid "TLS connection closed unexpectedly"
msgstr "اتصال TLS بطور غیر منتظره‌ای شکست خورد"

#: tls/gnutls/gtlsconnection-gnutls.c:474
#: tls/openssl/gtlsconnection-openssl.c:201
msgid "TLS connection peer did not send a certificate"
msgstr "همتا اتصال TLS گواهینامه‌ای ارسال نکرد"

#: tls/gnutls/gtlsconnection-gnutls.c:490
#, c-format
msgid "Peer sent fatal TLS alert: %s"
msgstr "همتا یک هشدارِ جدی TLS ارسال کرد: %s"

#: tls/gnutls/gtlsconnection-gnutls.c:500
msgid "Protocol version downgrade attack detected"
msgstr "حملهٔ پایین بردن نگارش قرارداد کشف شد"

#: tls/gnutls/gtlsconnection-gnutls.c:509
#, c-format
msgid "Message is too large for DTLS connection; maximum is %u byte"
msgid_plural "Message is too large for DTLS connection; maximum is %u bytes"
msgstr[0] "پیام برای اتّصال DTLS خیلی بزرگ است. بیشنه %Iu بایت است"
msgstr[1] "پیام برای اتّصال DTLS خیلی بزرگ است. بیشنه %Iu بایت است"

#: tls/gnutls/gtlsconnection-gnutls.c:518
msgid "The operation timed out"
msgstr "مهلت عملیات به سر رسید"

#: tls/gnutls/gtlsconnection-gnutls.c:870
#, c-format
msgid "Error performing TLS handshake: %s"
msgstr "خطا در هنگام انجام TLS handshake. خطا: %s"

#: tls/gnutls/gtlsconnection-gnutls.c:973
#: tls/openssl/gtlsconnection-openssl.c:618
msgid "Error performing TLS handshake"
msgstr "خطا در انجام دست دادن TLS"

#: tls/gnutls/gtlsconnection-gnutls.c:1152
#, c-format
msgid "Channel binding type is not implemented in the TLS library"
msgstr ""

#: tls/gnutls/gtlsconnection-gnutls.c:1156
#, c-format
msgid "Channel binding data is not yet available"
msgstr ""

#: tls/gnutls/gtlsconnection-gnutls.c:1194
#: tls/gnutls/gtlsconnection-gnutls.c:1206
#, c-format
msgid "X.509 certificate is not available on the connection"
msgstr ""

#: tls/gnutls/gtlsconnection-gnutls.c:1219
#, c-format
msgid "X.509 certificate is not available or is of unknown format: %s"
msgstr ""

#: tls/gnutls/gtlsconnection-gnutls.c:1230
#: tls/openssl/gtlsconnection-openssl.c:709
#, c-format
msgid "Unable to obtain certificate signature algorithm"
msgstr "ناتوان در دریافت الگوریتم امضای گواهی‌نامه"

#: tls/gnutls/gtlsconnection-gnutls.c:1246
#: tls/openssl/gtlsconnection-openssl.c:729
#, c-format
msgid ""
"Current X.509 certificate uses unknown or unsupported signature algorithm"
msgstr ""

#: tls/gnutls/gtlsconnection-gnutls.c:1340
#: tls/openssl/gtlsconnection-openssl.c:809
#, c-format
msgid "Requested channel binding type is not implemented"
msgstr ""

#: tls/gnutls/gtlsconnection-gnutls.c:1361
#: tls/gnutls/gtlsconnection-gnutls.c:1421
#: tls/openssl/gtlsconnection-openssl.c:827
#: tls/openssl/gtlsconnection-openssl.c:923
msgid "Error reading data from TLS socket"
msgstr "خطای خواندن داده از سوکت TLS"

#. flags
#: tls/gnutls/gtlsconnection-gnutls.c:1443
#: tls/gnutls/gtlsconnection-gnutls.c:1506
#: tls/openssl/gtlsconnection-openssl.c:1002
msgid "Error writing data to TLS socket"
msgstr "خطای نوشتن داده در سوکت TLS"

#: tls/gnutls/gtlsconnection-gnutls.c:1476
#, c-format
msgid "Message of size %lu byte is too large for DTLS connection"
msgid_plural "Message of size %lu bytes is too large for DTLS connection"
msgstr[0] "پیام با اندازهٔ %Ilu بایت برای اتّصال DTLS خیلی بزرگ است"
msgstr[1] "پیام با اندازهٔ %Ilu بایت برای اتّصال DTLS خیلی بزرگ است"

#: tls/gnutls/gtlsconnection-gnutls.c:1478
#, c-format
msgid "(maximum is %u byte)"
msgid_plural "(maximum is %u bytes)"
msgstr[0] "(بیشینه %Iu بایت است)"
msgstr[1] "(بیشینه %Iu بایت است)"

#: tls/gnutls/gtlsconnection-gnutls.c:1525
#, c-format
msgid "Error performing TLS close: %s"
msgstr "خطا در هنگام انجام بستن TLS. خطا: %s"

#: tls/gnutls/gtlsdatabase-gnutls.c:688
msgid ""
"Failed to load system trust store: GnuTLS was not configured with a system "
"trust"
msgstr ""
"شکست در بار کردن ذخیرهٔ اطمینان سامانه: GNUTLS با یک اطمینان سامانه پیکربندی "
"نشده بود"

#: tls/gnutls/gtlsdatabase-gnutls.c:693 tls/openssl/gtlsdatabase-openssl.c:229
#, c-format
msgid "Failed to load system trust store: %s"
msgstr "شکست در بار کردن ذخیرهٔ اطمینان سامانه: %s"

#: tls/gnutls/gtlsfiledatabase-gnutls.c:153
#: tls/openssl/gtlsfiledatabase-openssl.c:454
#, c-format
msgid "Failed to populate trust list from %s: %s"
msgstr "شکست در تکمیل سیاههٔ اطمینان از %s: %s"

#: tls/gnutls/gtlsserverconnection-gnutls.c:124
#: tls/openssl/gtlsserverconnection-openssl.c:172
#: tls/openssl/gtlsserverconnection-openssl.c:228
msgid "Certificate has no private key"
msgstr "گواهینامه هیچ کلید خصوصی‌ای ندارد"

#: tls/openssl/gtlsclientconnection-openssl.c:317
#: tls/openssl/gtlsserverconnection-openssl.c:346
#, c-format
msgid "Could not set TLS cipher list: %s"
msgstr "نتوانست سیاههٔ رمزنگاری TLS را تنظیم کند: %s"

#: tls/openssl/gtlsclientconnection-openssl.c:343
#: tls/openssl/gtlsserverconnection-openssl.c:372
#, c-format
msgid "Could not set MAX protocol to %d: %s"
msgstr "نتوانست شیوه‌نامهٔ MAX را به %Id تنظیم کند: %s"

#: tls/openssl/gtlsclientconnection-openssl.c:404
#: tls/openssl/gtlsserverconnection-openssl.c:433
#, c-format
msgid "Could not create TLS context: %s"
msgstr "ناتوان در ایجاد محتوای TLS: %s"

#: tls/openssl/gtlsconnection-openssl.c:217
#, c-format
msgid "Unacceptable TLS certificate authority"
msgstr "مرجع گواهینامهٔ TLS غیر قابل پذیرش"

#: tls/openssl/gtlsconnection-openssl.c:225
msgid "Digest too big for RSA key"
msgstr "دایجست برای کلید RSA خیلی بزرگ است"

#: tls/openssl/gtlsconnection-openssl.c:234
msgid "Secure renegotiation is disabled"
msgstr "مذاکرهٔ دوبارهٔ امن از کار افتاده است"

#: tls/openssl/gtlsconnection-openssl.c:253
#, c-format
msgid "%s: The connection is broken"
msgstr "%s: اتّصال خراب است"

#: tls/openssl/gtlsconnection-openssl.c:678
#, c-format
msgid "Channel binding data tls-unique is not available"
msgstr ""

#: tls/openssl/gtlsconnection-openssl.c:701
#, c-format
msgid "X.509 Certificate is not available on the connection"
msgstr ""

#: tls/openssl/gtlsconnection-openssl.c:747
#, c-format
msgid "Failed to generate X.509 certificate digest"
msgstr ""

#: tls/openssl/gtlsconnection-openssl.c:778
#, c-format
msgid "TLS Connection does not support TLS-Exporter feature"
msgstr ""

#: tls/openssl/gtlsconnection-openssl.c:781
#, c-format
msgid "Unexpected error while exporting keying data"
msgstr ""

#: tls/openssl/gtlsconnection-openssl.c:1063
msgid "Error performing TLS close"
msgstr "خطای انجام بستن TLS"

#: tls/openssl/gtlsdatabase-openssl.c:197
msgid "Could not get trusted anchors from Keychain"
msgstr ""

#: tls/openssl/gtlsdatabase-openssl.c:269
msgid "Could not create CA store"
msgstr "ناتوان در ایجاد ذخیرهٔ CA"

#: tls/openssl/gtlsserverconnection-openssl.c:179
#: tls/openssl/gtlsserverconnection-openssl.c:247
#, c-format
msgid "There is a problem with the certificate private key: %s"
msgstr "مشکلی با کلید خصوصی گواهینامه وجود دارد: %s"

#: tls/openssl/gtlsserverconnection-openssl.c:188
#: tls/openssl/gtlsserverconnection-openssl.c:239
#, c-format
msgid "There is a problem with the certificate: %s"
msgstr "مشکلی با گواهینامه وجود دارد: %s"

#~ msgid "Failed to load file path: %s"
#~ msgstr "شکست در بار کردن مسیر پرونده: %s"

#~ msgid "Operation would block"
#~ msgstr "عملیات می‌تواند بلوکه شود"

#~ msgid "Peer requested illegal TLS rehandshake"
#~ msgstr "همتا درخواست یک TLS rehandshake غیرقانونی کرده است"

#~ msgid "Server did not return a valid TLS certificate"
#~ msgstr "کارگزار گواهینامه TLS معتبری ارسال نکرد"

#~ msgid ""
#~ "This is the last chance to enter the PIN correctly before the token is "
#~ "locked."
#~ msgstr "آخرین شانس برای صحیح وارد کردن PIN قبل از قفل شدن رمز است."

#~ msgid ""
#~ "Several PIN attempts have been incorrect, and the token will be locked "
#~ "after further failures."
#~ msgstr ""
#~ "تعدادی از تلاش‌های برای وارد کردن PIN شکست خورده است، و رمز پس از شکست‌های "
#~ "بعدی قفل خواهد شد."

#~ msgid "The PIN entered is incorrect."
#~ msgstr "عبارت PIN وارد شده نادرست است."

#~ msgid "Module"
#~ msgstr "ماژول"

#~ msgid "PKCS#11 Module Pointer"
#~ msgstr "نشانگر ماژول PKCS#11"

#~ msgid "Slot ID"
#~ msgstr "شناسه‌ی جایگاه"

#~ msgid "PKCS#11 Slot Identifier"
#~ msgstr "شناساگر جایگاه PKCS#11"

#~ msgid "Connection is already closed"
#~ msgstr "اتصال از قبل بسته شده است"
